modules/acquia_cms_common/acquia_cms_common.info.yml

```diff
@@ -7,6 +7,7 @@ dependencies:
   - drupal:ckeditor
   - drupal:content_moderation
   - drupal:media
+  - drupal:media_library
```

Comment:
@vishalkhode1 @chandan-singh7929 Do you see any reason not to require media_library here? In order to set the filtered_html format config when the site studio module is installed, the filter.format.filtered_html.yml from the common module needs to be in place, which means it shouldn't be in optional config, but since there is a dependency on some media_library config in the filtered_html config, this module would need to require it. Normally, when the full profile is installed, place requires media_library, so that dependency is satisfied. Does this approach makes sense to you? The other option might be to dynamically create it on install of the common module if we don't want to add dependencies.

Reply:
This make sense to me, currently we have added dependency where it is require ex:
- acquia_cms_place.info.yml
- acquia_cms_place.info.yml

Maybe we can remove from above if we are going to provide from common, since they already require common as dependency.